### PR TITLE
feat: bundle dm-core using Rollup

### PR DESCRIPTION
**Pull request by eoaksnes**
## What does this pull request change?

* Create dist of dm-core 
* Set deps as peerDependencies
* Remove dm-core from yarn workspaces 
* Changed some of the references (from .../index to relative) used inside dm-core so that it was possible to bundle/build it
 
## Why is this pull request needed?

* Be able to include builed dm-core in applications

### CODE REVIEW

File: web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.test.tsx [hunk 1]
```diff
@@ -27,6 +28,7 @@ describe('AnalysisOverview', () => {
 
   describe('create analysis', () => {
     DOMAIN_ROLES.forEach((role: string) => {
+      console.log(role)
```
Comment (soofstad) [inline web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.test.tsx:31]:
```suggestion
```

File: web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.tsx [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 import { DEFAULT_DATASOURCE_ID } from '../../const'
 import { EBlueprints } from '../../Enums'
 import { TAnalysis } from '../../Types'
 import { AnalysisTable, CreateAnalysisButton } from './components'
 import { Divider } from '@equinor/eds-core-react'
+import { AuthContext } from 'react-oauth2-code-pkce'
 
 export const AnalysisOverview = (): JSX.Element => {
   const { tokenData } = useContext(AuthContext)
+  // @ts-ignore
```
Comment (soofstad) [inline web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.tsx:17]: No solution to this?

Comment (eoaksnes) [reply web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.tsx]: Removed :)

File: web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.test.tsx (hunk 1 above)
Comment (eoaksnes) [reply web/packages/plugins/analysis-platform/src/modules/Analysis/AnalysisOverview.test.tsx]: Removed :)

**Review by soofstad — APPROVED** [review]

**Pull request merged**